app.js

```diff
@@ -19,7 +19,33 @@ app.use(express.static(path.join(__dirname, 'public')));
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index')
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      let beers = [];
+      beersFromApi.forEach(beer => {
```

Comment:
You don't need to push the beers to a new array, you can also just use the beersFromApi-array.